tests/test.py

```diff
@@ -48,7 +53,7 @@ def test_server():
 def test_upload_success():
     with open(test_file, 'rb') as file:
         file_form = {'file': file}
-        r = requests.post(url=upload_url, files=file_form)
+        r = requests.post(url=upload_url, files=file_form, cookies=cookie)
```

Comment:
it seems excessive to add an argument to each test. is it necessary for all of them? they should still pass without one, right?

Reply:
I thought that too at first but for a test to see files uploaded in a previous test it needs the same cookie, therefore each test needs to set the cookie. This also allows tests to test user personas this way, there are two "users" in the tests: cookie and cookie2